src/index.html

```diff
@@ -20,8 +21,404 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="#home"
+        class="logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav-links">
+        <ul class="nav-items">
+          <li class="nav-item">
+            <a
+              href="#apple"
+              class="nav-link nav-link--is-active"
```

Comment:
```suggestion
              class="nav-link nav-link is-active"
```
Add just `is-active` as described (in one of the previous tasks, at least, about Header only). It's needed to pass all tests (test tries to find element with `is-active` class)